config/locales/pt-BR.yml

```diff
@@ -47,6 +47,7 @@ pt-BR:
       edit_post: Editar Post
       edit_private_topic: :thredded.nav.edit_topic
       edit_topic: Editar
+      mark_all_read: Marcar Como Lido
```

Comment:
@jayroh translated via google translate, this might say something horrible

obrigado!